### DISCUSSION TITLE
How to completely uninstall DBeaver on Mac

### DISCUSSION THREAD

**Discussion by jim256** [dbeaver/dbeaver#11316] [Q&A] [answered]
#### System information: 
- macOS Catalina 10.15.6 (19G73)
- DBeaver version: 7.1.5

#### Describe your question:

I want to completely remove everything DBeaver related and start over fresh with a brand new install that will not remember any of my preferences, cache, history, settings, connections, drivers, etc.

How can I do this on Mac? I've tried a few routes and after I reinstall, I can tell on opening it again that not everything got removed (it'll remember connection info, theme, etc.).

**Comment by serge-rider** [comment, marked as answer]
If you need to cleanup all your local files/settings then you can simple delete directory `~/Library/DBeaverData`.
Don't forget to close DBeaver before that.

**Comment by jim256** [comment]
@serge-rider , thank you. I think this worked. After I did that, rebooted, and re-installed, the only thing that was still the same was that it was using the dark theme. Is that now the default?

**Comment by serge-rider** [comment]
On MacOS dark theme is default if you use dark theme in MacOS itself.
In any case you can change it in the Preferences->Appearance.

**Comment by jim256** [comment]
Thanks. Closing.

**Comment by flatroof** [comment]
Are there any other code dependencies installed, e.g. java? or are they within the app package itself
